virt/install/uninstalling-virt.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="uninstalling-virt"]
+include::_attributes/common-attributes.adoc[]
+= Uninstalling {VirtProductName}
+:context: uninstalling-virt
+
+toc::[]
+
+You can uninstall {VirtProductName} by using the {product-title} xref:../../web_console/web-console.adoc#web-console-overview_web-console[web console] or the command line interface (xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[CLI]).
+
+[IMPORTANT]
+====
+You must delete all xref:../../virt/virtual_machines/virt-delete-vms.adoc#virt-delete-vm-web_virt-delete-vms[virtual machines], xref:../../virt/virtual_machines/virt-manage-vmis.adoc#virt-deleting-vmis-cli_virt-manage-vmis[virtual machine instances],
+and xref:../../virt/virtual_machines/virtual_disks/virt-deleting-datavolumes.adoc#virt-deleting-dvs_virt-deleting-datavolumes[data volumes] before you uninstall {VirtProductName}. If {VirtProductName} workloads exist, you cannot delete the `HyperConverged` custom resource (CR).
+====
+
+[id='uninstalling-virt-web-console_{context}']
+== Uninstalling {VirtProductName} using the web console
+
+You uninstall {VirtProductName} by deleting the `HyperConverged` CR, the {VirtProductName} Operator, and the `openshift-cnv` namespace.
+
+include::modules/virt-deleting-deployment-custom-resource.adoc[leveloffset=+2]
+
+Delete the {VirtProductName} Operator from the `openshift-cnv` namespace.
+
+include::modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc[leveloffset=+2]
+
+Delete the `openshift-cnv` namespace.
```

Comment:
Same comment here as for _Delete the {VirtProductName} Operator from the `openshift-cnv` namespace._